### DISCUSSION TITLE
[custom dataset] vits first success, slovak model 12000 steps (4 hours wallclock)

### DISCUSSION THREAD

**Discussion by neurlang** [coqui-ai/TTS#1602] [Show and tell]
Kanada šokovala Švédsko. V závere zmazala trojgólové manko a postúpila do semifinále

kɑnada ʃɔkɔvala ʃvɛːdskɔ. v zaːvɛrɛ zmɑzɑla trɔjgɔːlɔvɛː mankɔ a pɔstuːpilɑ dɔ sɛmifinaːlɛ.

https://user-images.githubusercontent.com/77860779/170557823-41f08544-c138-4ddf-9a97-bab3e78df24d.mp4

![image](https://user-images.githubusercontent.com/77860779/170557851-12bcc51b-6efc-4732-a218-87cdfedd36eb.png)

The dataset is still kind of small, and needs improving, but the model generalizes ok.

Phones in use for the model.
```
LETTER | PHONE
a	a
á	aː
ä	ɛ
b	b
c	ts
č	tʃ
d	d
ď	ɟ
dz	dz
dž	dʒ
e	ɛ
é	ɛː
f	f
g	g
h	h
ch	x
i	i
í	iː
j	j
k	k
l	l
ľ	ʎ
ĺ	lː
m	m
n	n
ň	ɲ
o	ɔ
ó	ɔː
ô	wɔ
p	p
q	kʋ
r	r
ŕ	rː
s	s
š	ʃ
t	t
ť	θ
u	u
ú	uː
v	v
w	v
x	ks
y	i
ý	iː
z	z
ž	ʒ
```

**Comment by erogol** [comment]
Nice!! Looking forward to see the end result.
